eppo/src/main/java/cloud/eppo/android/RuleEvaluator.java

```diff
@@ -48,21 +45,57 @@ private static boolean matchesRule(SubjectAttributes subjectAttributes, Targetin
         return !conditionEvaluations.contains(false);
     }
 
+
+
     private static boolean evaluateCondition(SubjectAttributes subjectAttributes, TargetingCondition condition
     ) {
         if (subjectAttributes.containsKey(condition.getAttribute())) {
             EppoValue value = subjectAttributes.get(condition.getAttribute());
+            Boolean isValueSemVer = Version.isValid(value.stringValue());
+            Boolean isConditionSemVer = Version.isValid(condition.getValue().stringValue());
+
             try {
                 switch (condition.getOperator()) {
                     case GreaterThanEqualTo:
-                        return Compare.compareNumber(value.doubleValue(), condition.getValue().doubleValue()
-                                , (a, b) -> a >= b);
+                        if (value.isNumeric() && condition.getValue().isNumeric()) {
```

Comment:
Instead to using `value.isNumeric() && condition.getValue().isNumeric()` can we create a boolean variable in the starting and reuse it everywhere. What do you think?